contracts/RMRK/base/RMRKBaseStorage.sol

```diff
@@ -77,15 +70,15 @@ contract RMRKBaseStorage is IRMRKBaseStorage {
      * @dev Returns symbol of associated collection
      * @return string base contract symbol
      */
-    function symbol() public view returns (string memory) {
-        return _symbol;
+    function getMetadataURI() external view returns (string memory) {
+        return _metadataURI;
     }
 
     /**
      * @dev Returns type of data of associated base
      * @return string data type
      */
-    function type_() public view returns (string memory) {
+    function getType() external view returns (string memory) {
         return _type;
     }
```

Comment:
We could just change the visibility of the `_type` variable to `public` and have the autogenerated getter. Wouldn't this reduce the file size?

Reply:
It does the opposite, increases file size

Reply:
Ok, then the point is moot 😄